art/attacks/inference/attribute_inference/baseline.py

```diff
@@ -90,10 +100,15 @@ def __init__(
         """
         super().__init__(estimator=None, attack_feature=attack_feature)
 
-        self._values: Optional[list] = None
+        self._values: list = []
```

Comment:
Empty lists should not be default values for arguments because if multiple instances of this class are created they will overwrite each others arguments.
```suggestion
        self._values: Optional[list] = None
```

Reply:
Please ignore.